README.rst

```diff
@@ -94,59 +124,73 @@ Or you can source the provided `init.sh` script which shall initialise terraform
    OpenStack Cloud Name: sms-lab
    Password:
 
-Generate Terraform variables:
+You must ensure that you have `Ansible installed <https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html>`_ on your local machine.
 
 .. code-block:: console
 
-   cat << EOF > terraform.tfvars
-   prefix = "changeme"
+   pip install --user ansible
 
-   ansible_control_vm_flavor = "general.v1.small"
-   ansible_control_vm_name   = "ansible-control"
-   ansible_control_disk_size = 100
+Install the Ansible galaxy requirements.
 
-   seed_vm_flavor = "general.v1.small"
-   seed_disk_size = 100
+.. code-block:: console
 
-   multinode_flavor     = "general.v1.medium"
-   multinode_image      = "Rocky9-lvm"
-   multinode_keypair    = "changeme"
-   multinode_vm_network = "stackhpc-ipv4-geneve"
-   multinode_vm_subnet  = "stackhpc-ipv4-geneve-subnet"
-   compute_count        = "2"
-   controller_count     = "3"
-   compute_disk_size    = 100
-   controller_disk_size = 100
+   ansible-galaxy install -r ansible/requirements.yml
 
-   ssh_public_key = "~/.ssh/changeme.pub"
-   ssh_user       = "cloud-user"
+If the deployed instances are behind an SSH bastion you must ensure that your SSH config is setup appropriately with a proxy jump.
 
-   storage_count     = "3"
-   storage_flavor    = "general.v1.small"
-   storage_disk_size = 100
+.. code-block:: console
```

Comment:
Not console, but also not really sure what this should be. Maybe just leave generic

Reply:
Done